### PR TITLE
Create 'UX-and-design-proposals.md'

### PR DESCRIPTION
**Pull request by clagom**
As discussed with the team, we want to start implementing these guidelines on how to share and comment the UX/design proposals across tools and disciplines, including a convention on how to provide feedback.

### CODE REVIEW

File: workflow/UX-and-design-proposals.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+  - [Links to specs/docs/epics](#links-to-specs-docs-epics)
+- [Who is it for?](#who-is-it-for)
+- [Inviting people to projects](#inviting-people-to-projects)
+- [When do we comment?](#when-do-we-comment)
+  - [Directions](#directions)
+  - [Does it need a comment?](#does-it-need-a-comment)
+- [Resolving comments](#resolving-comments)
+
+
+## feedback-convention
```
Comment (cristinadresch) [inline workflow/UX-and-design-proposals.md:29]: ## Feedback convention

File: workflow/UX-and-design-proposals.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+**2. I wish… (tag a yellow colour on the Zeplin comment)**
+For components that you think
+- Can be improved
+- Disagree with and would like to propose an alternative behaviour
+- Would like to patternized
+- Need clarification (express which part is confusing)
+- If you have a question, please formulate it as "I wish...", this will be really helpful to learn about what you have in mind.
+ 
+**3. INFO: (tag a purple colour on the Zeplin comment)**
+- When you don’t want to express an opinion, but would like to share information, caveat, or references that I might have missed or overlooked in the prototype/wireframe.
```
Comment (cristinadresch) [inline workflow/UX-and-design-proposals.md:47]: - When you don’t want to express an opinion, but would like to share information, caveat, or references that might have been missed or overlooked in the prototype/wireframe.

File: workflow/UX-and-design-proposals.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+- Feedback on the visual
+
+**Development**
+- This is ready to be implemented
+- Highlight implementation specs, annotation
+
+### Proposed tags, to be added for review stages
+We discussed reflecting the tag-naming convention from GitHub when asking for a certain type of feedback:
+- Review: UX needed
+- Review: design needed
```
Comment (cristinadresch) [inline workflow/UX-and-design-proposals.md:95]: Review: Design needed

File: workflow/UX-and-design-proposals.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+
+## Links to specs/docs/epics
+
+Do put links to specs, docs or issues/epics in the description field, so people can find their way back to the spec, and to the associated cards.
+
+
+### Who is it for?
+
+What feedback do you want? UX input, visual input, front-end input, engineering input etc.
+When uploading, think about the status of the project, what would you like feedback on may affect who you want to share it with it.
```
Comment (cristinadresch) [inline workflow/UX-and-design-proposals.md:124]: When uploading, think about the status of the project, what would you like feedback on may affect who you want to share it with.

File: workflow/UX-and-design-proposals.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+  - [Links to specs/docs/epics](#links-to-specs-docs-epics)
+- [Who is it for?](#who-is-it-for)
+- [Inviting people to projects](#inviting-people-to-projects)
+- [When do we comment?](#when-do-we-comment)
+  - [Directions](#directions)
+  - [Does it need a comment?](#does-it-need-a-comment)
+- [Resolving comments](#resolving-comments)
+
+
+## Feedback-convention
```
Comment (anthonydillon) [inline workflow/UX-and-design-proposals.md:29]: Can you remove `-`

File: workflow/UX-and-design-proposals.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+### Zeplin
+When ready for wireframes it can then move to Zeplin. Use tags and naming convention in a way that reflects the status of the project.
+
+### General direction
+We should work more on documents/specs to discuss things first, back and forth with engineers to discuss sketches about scope before moving to Zeplin for further feedback.
+
+
+## Zeplin/Miro status convention
+
+### Status & tags
```
Comment (anthonydillon) [inline workflow/UX-and-design-proposals.md:67]: Status and tags

**Comment by anthonydillon** [comment]
@clagom In future can you follow the contribution docs please: https://github.com/canonical-web-and-design/practices/blob/master/CONTRIBUTING.md#approval-criteria

**Comment by clagom** [comment]
Sorry @anthonydillon, missed it completely! 

"When you add, remove or move a document, please update the navigation structure contained in both _layouts/base.html and index.md." < Do I have to do anything here? I see it in the side nav already.

**Comment by anthonydillon** [comment]
@clagom no prob. no that is out of date as of https://github.com/canonical-web-and-design/practices/pull/124

**Comment by nottrobin** [comment]
Filed https://github.com/canonical-web-and-design/practices/pull/239

**Pull request merged**